x-pack/plugins/security_solution/server/endpoint/services/fleet/endpoint_fleet_services_factory.ts

```diff
@@ -41,35 +55,13 @@ export class EndpointFleetServicesFactory implements EndpointFleetServicesFactor
     return {
       agent: agentService.asInternalUser,
       agentPolicy,
+
```

Comment:
was this on purpose? :)

Reply:
yeah, it was... more for readability in separating related services